src/stores/breakoutRooms.js

```diff
@@ -0,0 +1,307 @@
+				this.processConversations(token, response.data.ocs.data)
+
+				// Get the participants of the breakout rooms
+				await this.fetchBreakoutRoomsParticipants(token)
+
+			} catch (error) {
+				console.error(error)
+				showError(t('spreed', 'An error occurred while re-ordering the attendees'))
+			}
+		},
+
+		/**
+		 * Deletes configured breakout rooms for a given parent room token.
+		 *
+		 * @param {string} token the parent room token.
+		 */
+		async deleteBreakoutRooms(token) {
+			try {
+				const response = await deleteBreakoutRooms(token)
+				// Update returned parent conversation
+				this.processConversations(token, response.data.ocs.data)
+				// Remove breakout rooms from this store
+				this.purgeBreakoutRoomsStore(token)
+			} catch (error) {
+				console.error(error)
+				showError(t('spreed', 'An error occurred while deleting breakout rooms'))
+			}
+		},
+
+		/**
+		 * Get configured breakout rooms for a given parent room token.
+		 *
+		 * @param {string} token the parent room token.
+		 */
+		async getBreakoutRooms(token) {
+			try {
+				const response = await getBreakoutRooms(token)
+				this.processConversations(token, response.data.ocs.data)
+			} catch (error) {
+				console.error(error)
+			}
+		},
+
+		/**
+		 * Start a breakout rooms session for a given parent room token.
+		 *
+		 * @param {string} token the parent room token.
+		 */
+		async startBreakoutRooms(token) {
+			try {
+				const response = await startBreakoutRooms(token)
+				this.processConversations(token, response.data.ocs.data)
+			} catch (error) {
+				console.error(error)
+				showError(t('spreed', 'An error occurred while starting breakout rooms'))
+			}
+		},
+
+		/**
+		 * Stop a breakout rooms session for a given parent room token.
+		 *
+		 * @param {string} token the parent room token.
+		 */
+		async stopBreakoutRooms(token) {
+			try {
+				const response = await stopBreakoutRooms(token)
+				this.processConversations(token, response.data.ocs.data)
+			} catch (error) {
+				console.error(error)
+				showError(t('spreed', 'An error occurred while stopping breakout rooms'))
+			}
+		},
+
+		/**
+		 * Send a message to all breakout rooms for a given parent room token.
+		 *
+		 * @param {object} payload the action payload;
+		 * @param {string} payload.token the parent room token;
+		 * @param {string} payload.message the message text.
+		 */
+		async broadcastMessageToBreakoutRooms({ token, message }) {
+			try {
+				await broadcastMessageToBreakoutRooms(token, message)
+			} catch (error) {
+				console.error(error)
+				showError(t('spreed', 'An error occurred while sending a message to the breakout rooms'))
+			}
+		},
+
+		/**
+		 * Update a participants in breakout rooms for a given token.
+		 *
+		 * @param {string} token the parent room token.
+		 */
+		async fetchBreakoutRoomsParticipants(token) {
+			try {
+				const response = await getBreakoutRoomsParticipants(token)
```

Comment:
```suggestion
				const response = await fetchBreakoutRoomsParticipants(token)
```

Rename this to `fetchBreakoutRoomsParticipants` ?